contrib/platform/test/com/sun/jna/platform/win32/AbstractWin32TestSupport.java

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2007 Timothy Wall, All Rights Reserved
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ */
+package com.sun.jna.platform.win32;
+
+import com.sun.jna.platform.AbstractPlatformTestSupport;
+
+/**
+ * @author lgoldstein
+ */
+public abstract class AbstractWin32TestSupport extends AbstractPlatformTestSupport {
```

Comment:
I am not a huge fan of this, primarily because new test classes do things very differently, and you're not retrofitting all tests into this. So we now have several test classes written in different ways, some explicitly check, some with `assertTestSucceeded`, etc. 

Generally let me suggest something like this, borrowed from a convention I've seen for years in a bunch of places. 

```
checkBool('Kernel32.INSTANCE.GetNamedPipe...', "GetNamedPipe");
```

This way you can add similar functions like `checkHR` and others.

I would vote not to do this though unless you want to rewrite *all* tests to use this.